campaignresourcecentre/paragon/data_classes.py

```diff
@@ -93,6 +99,10 @@ def params(self):
             result["ProductRegistrationVar9"] = self.postcode
         if self.created_at:
             result["ProductRegistrationVar10"] = self.created_at
+        if self.postcode_region:
+            result["ContactVar3"] = self.postcode_region
```

Comment:
The Jira says to put the postcode in `ContactVar3` and region into `ProductRegistrationVar9` but I am guessing that requirement is wrong based on what you have seen the preference centre? If so are you OK to update the Jira please?

Reply:
If they don't want it/used then it would be wonderful to bin it off. Let's see what they say, but ye thanks for fixing the Jira